### DISCUSSION TITLE
Welcome to gspt Discussions!

### DISCUSSION THREAD

**Discussion by akurdunkar** [akurdunkar/gspt#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by RichardZhang104** [comment]
I love this cli, it's so aesthetic. Is there a way to hide the album art bit? I'm on macos and album art doesn't work.

**Reply by akurdunkar** [reply]
I think it's about time to add this option. Thanks for the suggestion I will add this feature soon.

**Reply by akurdunkar** [reply]
Fixed #8
